src/components/navigation-fab.tsx

```diff
@@ -0,0 +1,34 @@
+import { Pressable, StyleSheet, Text } from "react-native";
+import { Navigation } from "react-native-navigation";
+
+interface Props {
+  title: string;
+  srcId: string;
+  destName: string;
+}
+
+export function NavigationFAB({title, srcId, destName}: Props) {
+  
+  const onPress = () => Navigation.push(srcId, { component: { name: destName } });
```

Comment:
```suggestion
  const handlePress = () => Navigation.push(srcId, { component: { name: destName } });
```

Só uma questão de convenção nossa mesmo. O React já tem essa convenção do prefixo `on` para props que são callbacks, como o `onPress`. Então a gente adotou essa do `handle` para ser o nome da função que implementa algum callback. Vai ser comum vc ver o parzinho `onAlgumacoisa={handleAlgumaCoisa}`.